<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_RefreshProcesses">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh Process List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CheckProcessList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Process List</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ipAddress">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>console ip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ConnectPS4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_GetProcessInfo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Process Info</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_processName">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>process name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AttachProcess">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Attach Process</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NotifyText">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>text for notify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TestNotify">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Notify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TestProcMaps">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Process Maps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TestWrite">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Writing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TestReading">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Reading</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TestAllocation">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Allocating</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TestFreeing">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Freeing</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
